### ISSUE TITLE
Umbraco Delivery API not valid OpenAPI specification (according to APIM)

### ISSUE THREAD

**Issue by RBerntsen** [umbraco/Umbraco-CMS#17300]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

14.3.0

### Bug summary

When we try to import the JSON specification for the API's from Swagger (out of the box on v14) into Azure API Management, we get two errors:
1. The security scheme "Backoffice User" is invalid (cannot have a space)
![image](https://github.com/user-attachments/assets/ab797c91-f304-4b51-a910-9dd080bb328d)

2. Four operations with duplicate signatures:
![image](https://github.com/user-attachments/assets/5793bb3c-ab0f-4c2a-9cda-050833e608c2)

This is due to the fact that we have four pairs of endpoints that are somewhat equal, only difference is the name of the parameter, i.e. "path" and "id" are both of type string. 

The endpoints are these four pairs:
```
/umbraco/delivery/api/v1/content/item/{path}
/umbraco/delivery/api/v1/content/item/{id}

/umbraco/delivery/api/v2/content/item/{path}
/umbraco/delivery/api/v2/content/item/{id}

/umbraco/delivery/api/v1/media/item/{path}
/umbraco/delivery/api/v1/media/item/{id}

/umbraco/delivery/api/v2/media/item/{path}
/umbraco/delivery/api/v2/media/item/{id}
```

I think it would be relatively easy to rename the scheme for the "Backoffice User" to e.g. "BackofficeUser" as that looks to just be a const string:
![image](https://github.com/user-attachments/assets/4025f2a5-6916-4d6a-9192-09a7852dc6e5)

I'm unsure how exactly to handle the issue with the duplicate endpoints, as we would probably like both to be displayed in Swagger?

I have encountered similar issues on v12 and v13 as well.

### Specifics

See above

### Steps to reproduce

Navigate to your APIM instance in Azure.
Select "APIs" and select "OpenAPI" under the "Create from definition".

![image](https://github.com/user-attachments/assets/98685928-2720-489b-945b-0f2e702e1dc4)

From here you can upload the JSON file that was generated for the Swagger documentation. 

When doing this you will first be presented with the error for the security scheme. After fixing that you will be presented with the issue for the duplicate endpoints.

### Expected result / actual result

I would expect that the JSON file contained a valid OpenAPI specification that could be imported into APIM without modifications

---
_This item has been added to our backlog AB#45094_

**Comment by iOvergaard** [comment]
Hi @RBerntsen, thanks for the report. We will take a look at it in an upcoming sprint!

**Comment by kjac** [comment]
Hi @RBerntsen,

Thanks for reporting! The Delivery API OpenAPI spec should not have the Management API security scheme to begin with 🙈 

#17401 fixes the validation errors of the spec.

Do note that the "duplicate signatures" issue you're encountering is not caused by an invalid OpenAPI spec. The spec has explicit and unique operation IDs for each endpoint 👍

**Comment by elit0451** [comment]
Fixed in #17401
